sycl/test/abi/sycl_symbols_linux.dump

```diff
@@ -3784,6 +3789,10 @@ _ZN2cl4sycl6kernelC1EP10_cl_kernelRKNS0_7contextE
 _ZN2cl4sycl6kernelC1ESt10shared_ptrINS0_6detail11kernel_implEE
 _ZN2cl4sycl6kernelC2EP10_cl_kernelRKNS0_7contextE
 _ZN2cl4sycl6kernelC2ESt10shared_ptrINS0_6detail11kernel_implEE
+_ZN2cl4sycl6level010make_queueERKNS0_7contextEm
```

Comment:
Probably we should sort symbols to make it easier to understand what symbols were changed between releases.

Reply:
The script sorts symbols:
https://github.com/intel/llvm/blob/13a1e49eeb2e51c6d0b90ba4ee39dc33dd6bf640/sycl/tools/abi_check.py#L53

Developers, however, not necessarily use it. This is why I also updated header to discourage people from editing these files manually.